src/common/components/TxLink.tsx

```diff
@@ -0,0 +1,28 @@
+import { Link } from '@chakra-ui/next-js';
+import { useColorMode } from '@chakra-ui/react';
+import { FC } from 'react';
+
+import { Box, BoxProps } from '../../ui/Box';
+
+export const TxLink: FC<{ href?: string; openInNewTab?: boolean } & BoxProps> = ({
```

Comment:
Can [this component ](https://github.com/hirosystems/explorer/blob/5ab9f9545ae65195ef2784ea0906ede0ec990647/src/common/components/ExplorerLinks.tsx#L15)work for your case instead of creating a new one?